mbed_greentea/mbed_test_api.py

```diff
@@ -276,7 +276,11 @@ def get_auto_property_value(property_name, line):
         proc = Popen(cmd, stdin=PIPE, stdout=PIPE)
         obs = ProcessObserver(proc)
 
-    result = None
+    RE_TESTCASE_START = re.compile("^\{\{(testcase_start);(\w+)\}")
+    RE_TESTCASE_FINISH = re.compile("^\{\{(testcase_finish);(\w+);(\d+)\}\}")
+
+    result = None           # Test suite result
+    result_test_cases = {}  # Test cases results
     update_once_flag = {}   # Stores flags checking if some auto-parameter was already set
     unknown_property_count = 0
     total_duration = 20     # This for flashing, reset and other serial port operations
```

Comment:
Good point, I need to check that. Total duration should be for whole test suite execution time (for whole binary execution including flashing and reset).